packages/terra-menu/src/MenuItem.jsx

```diff
@@ -224,9 +226,13 @@ class MenuItem extends React.Component {
     }
 
     return (
-      <li {...attributes} className={itemClassNames} ref={this.setItemNode} role={role} aria-checked={markAsSelected}>
-        {content}
-      </li>
+      <ThemeContext.Consumer>
```

Comment:
I'm not familiar with contextTypes, but is this something that could be provided via the contextTypes like `isGroupItem` and `isSelectableMenu`?

Reply:
You can only read one context at a time with contextTypes. Since it's taken, we need to wrap.